rules/order.js

```diff
@@ -59,18 +107,59 @@ module.exports = {
       order: 'strict',
       properties: [
         'background',
+        'background-attachment',
+        'background-clip',
         'background-color',
-        'background-size',
-        'background-position',
+        'background-image',
         'background-repeat',
+        'background-position',
+        'background-size',
         'border',
         'border-top',
         'border-right',
         'border-bottom',
         'border-left',
+        'border-width',
+        'border-top-width',
+        'border-right-width',
+        'border-bottom-width',
+        'border-left-width',
+        'border-style',
+        'border-top-style',
+        'border-right-style',
+        'border-bottom-style',
+        'border-left-style',
+        'border-radius',
+        'border-top-left-radius',
+        'border-top-right-radius',
+        'border-bottom-left-radius',
+        'border-bottom-right-radius',
```

Comment:
```
'border-top-left-radius',
'border-top-right-radius',
'border-bottom-left-radius',
'border-bottom-right-radius',
```

^ This feels inconsistent to me, because the `border-width` and `-style` properties follow a clockwise rotation (top right bottom left) and these don't because `bottom-left` is before `bottom-right`.